om.lic

```diff
@@ -0,0 +1,82 @@
+
+custom_require.call(%w[common common-arcana common-travel])
+
+class CreateOsrelMeraud
+  include DRC
+  include DRCT
+
+  def initialize
+    unless DRStats.cleric?
+      echo "You're not a cleric!"
+      exit
+    end
+      
+    if DRSpells.active_spells.include?('Osrel Meraud')
+      echo "OM already active!"
+      exit
+    end
+
+    @settings = get_settings
+
+    unless @settings.waggle_sets['create_om']
+      echo "This script uses waggle_sets, you need a create_om. It will also buff with a buff_om set."
+      exit
+    end
+
+    echo "No OM found!"
+    
+    buff_om if create_om if get_orb
+  end
+
+  def get_orb
+    case bput("tap my #{@settings.osrel_favor_god} orb", 'I could not find', 'The orb is delicate')
```

Comment:
💡 Consider using `DRCI.exists?` instead of `bput("tap....`

Reply:
Good call, done!